app/code/community/Bolt/Boltpay/Model/Payment.php

```diff
@@ -1071,4 +1068,95 @@ protected function isPartialRefundFixingMismatch(Mage_Sales_Model_Order $order,
 
         return false;
     }
+
+    /**
+     * Whether this method can accept or deny payment
+     *
+     * @param Mage_Payment_Model_Info $payment
+     *
+     * @return bool
+     */
+    public function canReviewPayment(Mage_Payment_Model_Info $payment)
+    {
+       return $payment->getAdditionalInformation('bolt_transaction_status') == self::TRANSACTION_REJECTED_REVERSIBLE;
+    }
+
+    /**
+     * Attempt to accept a payment that us under review
+     *
+     * @param Mage_Payment_Model_Info $payment
+     * @return bool
+     * @throws Exception
+     */
+    public function acceptPayment(Mage_Payment_Model_Info $payment)
```

Comment:
let's call it approve instead of accept

Reply:
@daisy1754 @matt-thomason This is a superclass function that he is implementing here.  He could use a wrapper method to change the name, but for built in Magento, this is how they defined it. :(